build.sbt

```diff
@@ -51,7 +50,7 @@ ThisBuild / version := {
 }
 
 sbtPlugin := true
-pluginCrossBuild / sbtVersion := "1.5.5"
+pluginCrossBuild / sbtVersion := "1.9.5"
```

Comment:
So theoritically speaking I don't think you need to should need to update this because afaik this sbt setting is mandating the minimum version that the plugin needs to compile where as `build.properties` (which this PR adds/updats) specifies the actual sbt version that is used at runtime when the tests are being run.

@eed3si9n Maybe you have a better idea what is going on here? I also think that updating `pluginCrossBuild / sbtVersion` can be problematic because you are essentially forcing the minimum sbt version on downstream consumers and this sbt plugin does not actually need such a new sbt version.